Refresh.GameServer/Importing/Mip/MipMetadata.cs

```diff
@@ -0,0 +1,6 @@
+namespace Refresh.GameServer.Importing.Mip;
+
+public class MipMetadata
+{
+    
+}
```

Comment:
?

Reply:
This is an ImageSharp thing, it wants a metadata class, but since we have nothing to fill it out with, its left empty, you can see the reference [here](https://github.com/LittleBigRefresh/Refresh/pull/306/files#diff-8e2e55d3c11ef30dcab4a28cc7ed1bfa8ee1d5374e7e96e0e2ef3bcf729472d3)